### DISCUSSION TITLE
Long type hints in a function definition can cause the scaling to be slighly off.

### DISCUSSION THREAD

**Discussion by JackAshwell11** [pradyunsg/furo#297] [Potential Issue] [answered]
So this is specifically with https://lavapy.readthedocs.io/en/latest/lavapy.html#lavapy.NodePool.createNode. When viewing on a phone screen, the `discord.ext.commands.bot.AutoShardedBot` type in the function definition causes you to have to swipe the screen to the left to view the entire definition.

This could probably be fixed with a simple wrap around or scaling tweak although I'm not sure.

**Comment by pradyunsg** [comment]
It seems to be just fine for me. Could you include the following bits of information in a separate comment here:

- A screenshot showing the issue.
- Browser under use.
- OS version.
- Screen resolution or Phone model.

**Reply by JackAshwell11** [reply]
> It seems to be just fine for me. Could you include the following bits of information in a separate comment here:
> 
> - A screenshot showing the issue.
> - Browser under use.
> - OS version.
> - Screen resolution or Phone model.
> 

I'm using android 11 and chrome 95.0.4638.74 with a resolution of 384x854.

The two pictures show the default position and the position after swiping to the left.

![Screenshot_20211110-085907.jpg](https://user-images.githubusercontent.com/19552279/141082605-117e2164-61c3-4a40-a9df-2183cb1f2adb.jpg)

![Screenshot_20211110-085902.jpg](https://user-images.githubusercontent.com/19552279/141082626-9854c269-1263-4fca-8260-f7c1616d569b.jpg)


May just be a problem with sphinx but I just thought I'd bring it up.

**Comment by pradyunsg** [comment, marked as answer]
This was fixed: https://pradyunsg.me/furo/changelog/#hearty-honeydew
